README.md

```diff
@@ -148,3 +148,12 @@ RUN mkdir /etc/static-ngeo/web-components \
     && cp -r /usr/lib/node_modules/luxembourg-geoportail/bundle/assets/ /etc/static-ngeo/web-components/assets/ \
     && cp /usr/lib/node_modules/luxembourg-geoportail/bundle/style.css /etc/static-ngeo/web-components/style.css
 ```
+
+### Use components, services, stores, composables
+
+To use components, services, stores or composables in you application, all these should explicitely exported in the lib.
+
+To do so,
```

Comment:
To create custom components in the application using the lib, adapt the following code snippet to relevant components in the target app.